configs/service.yaml

```diff
@@ -15,6 +15,13 @@ redis_max_retry_backoff: "512ms"
 
 blob_max_inline_size: 65536
 
+grpc_keepalive_enabled: false
```

Comment:
Please remove this setting (grpc_keepalive_enabled) and always use the default values as configured in the service.yaml file when creating the gRPC server.
The yaml file already provides a way to set the values with defaults that make sense for whatever use case.